cluster/examples/kubernetes/ceph/common-external.yaml

```
@@ -35,7 +35,7 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: rook-ceph-system
-  namespace: rook-ceph
+  namespace: rook-ceph # namespace:system
```

How about this tag instead? I never did like how we called it the system namespace. It's really just the namespace where the operator is running.
```suggestion
  namespace: rook-ceph # namespace:operator
```